mobile/deviceTheme.js

```diff
@@ -75,18 +75,18 @@
 		//
 		//		In this case, the following files will be loaded.
 		//
-		//	|	dojox/mobile/themes/iphone/base.css
+		//	|	dojox/mobile/themes/ios7/base.css
 		//	|	com/acme/themes/iphone/MyWidget.css
```

Comment:
`iphone` => `ios7`  here too (as at line 89), isn't it?
(No need to update the PR, I can add it when merging.)